userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -36,6 +41,8 @@ Billing Period:: The frequency in which Kill Bill invoices the account for one o
 
 [[Bundle]]Bundle:: A collection of subscriptions associated with a single service instance. For example, a customer might have a base subscription for her phone, plus an add-on subscription for international dialing. Because both subscriptions are associated with the same phone, they are collected into a single bundle. An account may have several bundles, each with one or more subscriptions. Also known as a "subscription bundle."
 
+Bill Cycle Day:: The day of the month the invoice is created for an account. Note that this is applicable only to monthly and quarterly subscriptions.
```

Comment:
`monthly and quarterly subscriptions.` -> to monthly based subscriptions (`MONTHLY`, `QUARTERLY`, `ANNUAL`, `BIMESTRIAL`, `BIANNUAL`, `BIENNIAL`) as opposed to (`DAILY`, `THIRTY_DAYS`, `WEEKLY`, ..)

Reply:
Fixed